packages/foreman/nodejs-theforeman-vendor/nodejs-theforeman-vendor.spec

```diff
@@ -0,0 +1,592 @@
+%global npm_name @theforeman/vendor
+%global release 1
+%global prerelease beta.5
+
+Name: nodejs-theforeman-vendor
+Version: 0.1.0
+Release: %{?prerelease:0.}%{release}%{?prerelease:.}%{?prerelease}%{?nightly}%{?dist}
+Summary: foreman supported 3rd-party node_modules
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/sharvit/foreman-vendor#readme
+Source0: https://registry.npmjs.org/@theforeman/vendor/-/@theforeman/vendor-0.1.0-beta.5.tgz
+Source1: https://registry.npmjs.org/classnames/-/classnames-2.2.6.tgz
```

Comment:
I don't understand the question. How can you build without sources? Remember that our koji builder instance has no internet access and this guarantees reproducable builds. This is how we avoid leftpad.

Reply:
because the build of this rpm package is only copying a few files from the node module. 
The actual webpack compilation of the dist file is done separately when pushing a new version to npm, providing a file that is identical every time you install e.g. `@theforeman/vendor` version 0.1.0 - it will never change as it is part of the module on npm.

Reply:
In that case the vendor package should not declare them as dependencies I think but rather build dependencies.

https://docs.npmjs.com/files/package.json#dependencies
> Please do not put test harnesses or transpilers in your dependencies object. See devDependencies, below.